### ISSUE TITLE
Is the keras model's constructor placed in a wrong place?

### ISSUE THREAD

**Issue by yuboona** [yashsmehta/personality-prediction#8]
I found that, your code placed the `model=tf.keras.model.sequential` inside the K-fold level. I really think it results that it will generate too many new model every time your code go into K-fold, rather than one model for one **traits**.

**Comment by warrior-yyyan** [comment]
> I found that, your code placed the `model=tf.keras.model.sequential` inside the K-fold level. I really think it results that it will generate too many new model every time your code go into K-fold, rather than one model for one **traits**.

In my opinion, the code is correct. Every trait has 2467 samples, then use K-fold to divide 2467 samples into K datasets, K-1 as train dataset, and the rest one as test dataset. Through K times training, we could get K models for one trait, then get average acc or loss from K models' results.

**Comment by warrior-yyyan** [comment]
> I found that, your code placed the `model=tf.keras.model.sequential` inside the K-fold level. I really think it results that it will generate too many new model every time your code go into K-fold, rather than one model for one **traits**.

In my opinion, the code is correct. Every trait has 2467 samples, then use K-fold to divide 2467 samples into K datasets, K-1 as train dataset, and the rest one as test dataset. Through K times training, we could get K models for one trait, then get average acc or loss from K models' results.

**Comment by yashsmehta** [comment]
Thanks @warrior-yyyan!
